mypyc/genops.py

```diff
@@ -3900,15 +3949,24 @@ def unbox_or_cast(self, src: Value, target_type: RType, line: int) -> Value:
     def box_expr(self, expr: Expression) -> Value:
         return self.box(self.accept(expr))
 
-    def make_dict(self, key_value_pairs: List[Tuple[Value, Value]], line: int) -> Value:
+    def make_dict(self, key_value_pairs: List[KeyDatum], line: int) -> Value:
```

Comment:
Nit: Change this List to a Sequence so that callers could use it with a `List[Tuple[Value, Value]]` also.